packages/react-core/src/components/Page/PageGroup.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Page/page';
+
+export interface PageGroupProps extends React.HTMLProps<HTMLDivElement> {
+  /** Additional classes to apply to the PageGroup */
+  className?: string;
+  /** Content rendered inside of the PageGroup */
+  children?: React.ReactNode;
+  /** Modifier indicating if PageGroup is sticky to the top or bottom */
+  sticky?: 'top' | 'bottom';
+  /** Modifier indicating if PageGroup should have a shadow at the top */
+  hasShadowTop?: boolean;
```

Comment:
sticky top adds shadow bottom and sticky bottom adds shadow top by default right? but then you can’t disable it? then the prop name and/or description could be confusing since it seems you should be able to disable it.
It’s really not a modifier toggle, just an add, no?
so instead of `hasShadowBottom` should it be `addShadowBottom` and the same for the top?

Reply:
I don't know, maybe it's fine, it was just confusing to me in the moment. Perhaps just a description change to indicate that it is enabled by default for sticky?

Reply:
IMO the way it is now is fine. We use similar props for `pf-m-padding` and `pf-m-no-padding` to add padding to something that doesn't have it by default, and remove it from something that has it by default. I think the shadows behave the same here.

The way we handle those props in the drawer is `hasPadding` and `hasNoPadding`

https://github.com/patternfly/patternfly-react/blob/55e07e690fb8aa26288f3294f13d5819beaa961d/packages/react-core/src/components/Drawer/DrawerContentBody.tsx#L11

https://github.com/patternfly/patternfly-react/blob/55e07e690fb8aa26288f3294f13d5819beaa961d/packages/react-core/src/components/Drawer/DrawerPanelBody.tsx#L11

Reply:
You can see the use case for adding shadows to page sections here - https://pf4.patternfly.org/components/page/html-demos/overflow-scroll/

The top and bottom sections have shadows but aren't sticky. That page is 3 regular page sections, with the middle one is set to `pf-m-overflow-scroll`, meaning it enables a scrollbar for overflow content.